index.html

```diff
@@ -299,6 +299,18 @@ <h4>Game streaming</h4>
            delay. This requirement is addressed by jitterBufferTarget, defined in
            [[?WebRTC-Extensions]] Section 6.</td>
         </tr>
+        <tr>
+           <td>N48</td>
+           <td>The application must be able to control video decoding to continue even
+           after a frame-loss without waiting for a key frame. This helps the application recover
+           faster from lossy network conditions.</td>
+	</tr>
+	<tr>
+           <td>N49</td>
+           <td>To adapt the video quality to the network fluctuations and maintain
+           a steady latency, the application must have the ability to control the feedback
+           interval for packet loss information.</td>
```

Comment:
I'll repeat here what I said on the WG call:
The feedback interval is a method, not a requirement.
The requirement is that changes in the available bandwidth can be reacted to within <some interval, configurable> - the reaction can be a change in codec configuration (internal, not app-visible unless watching stats) or some kind of reporting to the application.

We should be writing requirements, not implementation descriptions.